includes/class-content-toc.php

```diff
@@ -118,11 +138,12 @@ public function shortcode( $atts, $content = null ) {
 				$item_text = strip_tags( $item_match_arr[1] );
 
 				// Add filter to allow custom TOC item markup
-				$toc_list .= apply_filters(
+				$items_html .= apply_filters(
```

Comment:
Thanks @sanchothefat for the feedback :) I've kept this filter and removed the other one. Could you let me know if that's ok to merge the code? Does it look alright?